cumulus_library/study_parser.py

```diff
@@ -179,20 +183,29 @@ def clean_study(
         # We'll do a pass to see if any of these tables were created outside of a
         # study builder, and remove them from the list.
         for view_table in view_table_list.copy():
-            if any(
-                view_table[0].startswith(f"{self.get_study_prefix()}__{word}_")
-                for word in RESERVED_TABLE_KEYWORDS
-            ):
+            if any((f"_{word}_") in view_table[0] for word in RESERVED_TABLE_KEYWORDS):
```

Comment:
So harsh! No more `covid_study__this_is_my_nlp_table`?

We might want to also catch the word at the end (which we didn't do before either) -- `prefix__nlp` or `prefix__this_is_nlp`)

Reply:
Also, maybe add a test for this new restriction.

Reply:
we had some of those in the historical stuff i was cleaning up - that's the source of this change, but i think i stand by it if the reserved with is small. can add a endswith check and a test for this.

Reply:
Yeah, easier to be strict than not